taui/src/locales/zh/translations.js

```diff
@@ -109,89 +115,88 @@ export default {
-    ImportanceViolentCrime: 'Public safety',
+    Destinations: '您最常去什麼地方（除了您的家)?',
+    ImportanceAccessibility: '通勤時間',
+    ImportanceHeading: '選擇住處的時候以下因素有多重要？',
+    ImportanceSchools: '學校質量',
+    ImportanceViolentCrime: '公共安全',
     Primary: 'Primary',
-    Purpose: 'Purpose',
-    RoomsVoucher: 'Number of bedrooms on your voucher',
+    Purpose: '目的',
+    Rooms: '租賃券上的臥房數目',
     RoomsNoVoucher: 'Desired number of bedrooms',
     Budget: 'Budget for apartment',
     YesVoucher: 'Yes',
     NoVoucher: 'No',
-    Go: 'Save profile',
+    Go: '保存文檔',
     NameRequired: 'Please enter a name for the head of household.',
     SaveError: 'Failed to save profile. Please try again.',
     InvalidBudget: 'Invalid budget. Please enter a positive number.',
-    Title: 'Profile',
-    UseCommuterRail: 'I’m willing to take the express bus or commuter rail',
-    UseCommuterRailExplanation: 'The Commuter rail and express bus allow us to recommend more neighborhoods, but they usually cost more than the subway or local bus.'
+    Title: '文檔',
+    UseCommuterRail: '包括通勤火車和特快巴士',
+    UseCommuterRailExplanation: '搜尋結果將包括地鐵、本地巴士、通勤火車和特快巴士。通勤火車和特快巴士將可到達更多的社區，但其費用通常比地鐵和本地巴士服務要高。'
   },
   Tooltips: {
-    AboveAverage: 'above average',
-    Average: 'about average',
-    BelowAverage: 'below average',
-    Education: '<p><strong>{{town}}</strong> schools are <strong>{{averageRelation}}</strong> for the state and rank higher than <strong>{{edPercentile}}</strong> of other school districts when considering overall school quality and performance of low-income students. (Source: <a href="http://profiles.doe.mass.edu/" target="_blank">MA DOE</a>)</p>',
-    ExpandedChoice: '<p>Expanded Choice communities are TK</p>',
-    ViolentCrime: '<p><strong>{{town}}</strong> has <strong>{{averageRelation}}</strong> public safety, safer than <strong>{{crimePercentile}}</strong> of other zipcodes in the state. (Source: <a href="https://www.fbi.gov/services/cjis/ucr" target="_blank">FBI Uniform Crime Reporting violent crime data</a>)</p>',
-    ViolentCrimeBoston: '<p><strong>{{town}}</strong> has <strong>{{averageRelation}}</strong> public safety, safer than <strong>{{crimePercentile}}</strong> of other zipcodes in the state. (Source: <a href="https://data.boston.gov/" target="_blank">Boston Police Department Incident Reports</a>)</p>',
-    RentalUnits: '<p><strong>{{town}}</strong> has an <strong>{{averageRelation}}</strong> number of rental units for the state. From 2017-2018 there were <strong>{{totalMapc}}</strong> units listed on Craigslist. (Source: <a href="https://www.mapc.org/" target="_blank">MAPC</a>)</p>'
+    AboveAverage: '高於平均水平',
+    Average: '相對平均水平',
+    BelowAverage: '低於平均水平',
+    Education: '<p>在參考過學校的整體質量和低收入家庭學生的表現，（如有低收入家庭學生表現的數據可使用），<strong>{{town}}</strong> 的學校在本州 <strong>{{averageRelation}}</strong> ，其排名高於[ ]的其他學區 <strong>{{edPercentile}}</strong> (資料來源: <a href="http://profiles.doe.mass.edu/" target="_blank">馬薩諸塞州中小學教育部 MA DESE</a>)</p>',
```

Comment:
Yeah, I also noticed this and thought it was strange so thanks for catching it. I also agree it should be changed to DESE because the app uses DOE to refer to the US Dept of Energy in other places.